web/docs/tql2/operators/legacy.md

```diff
@@ -1,8 +1,14 @@
 # legacy
 
-:::warning
-This operator will be removed once TQL2 is stable and all TQL1 functionality is
-ported over.
+:::warning Migrate to TQL2
+TQL1 is deprecated and will be removed with the upcoming Tenzir Node v5.0
+release, which will make TQL2 the default.
+
+Currently, TQL2 is opt-in. You probably got to this page because you've seen a
+warning in your TQL1 pipelines that pointed you to this page.
+
+To prepare for this migration and use TQL2 for all pipelines today, we recommend
+setting the environment variable `TENZIR_TQL2=true`.
```

Comment:
Since this is in the docs of `legacy`, I feel like this should spell out more what will happen to `legacy`. For example, the `TENZIR_TQL2=true` flag is only slightly related to the operator itself. The next steps would be: warning, change-default, enforce-tql2-default-but-keep-legacy, delete-legacy. Where do we land on this scale?